File: javascript-closure-angular-client/API/Client/PurchaseInstallmentIn.js
goog.provide('API.Client.PurchaseInstallmentIn');

/**
 * @record
 */
API.Client.PurchaseInstallmentIn = function() {}

/**
 * DOCNUMBER, ID OF THIS DOCUMENT FOR THIS transaction
 * @type {!string}
 * @export
 */
API.Client.PurchaseInstallmentIn.prototype.documentNumber;

/**
 * installment, Due Date
 * @type {!API.Client.date}
 * @export
 */
API.Client.PurchaseInstallmentIn.prototype.date;

/**
 * Intallment value, fraction of  ∑(lineAmount - lineTaxedDiscount)
 * @type {!number}
 * @export
 */
API.Client.PurchaseInstallmentIn.prototype.grossValue;

File: async-scala-client/src/main/scala/io/swagger/client/model/PurchaseInstallmentIn.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class PurchaseInstallmentIn (
  documentNumber: String,  // DOCNUMBER, ID OF THIS DOCUMENT FOR THIS transaction
date: Date,  // installment, Due Date
grossValue: Double  // Intallment value, fraction of  ∑(lineAmount - lineTaxedDiscount)
)
